[2025-03-17T17:47:39.985+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gen_data_daily.get_date scheduled__2025-03-14T22:00:00+00:00 [queued]>
[2025-03-17T17:47:39.996+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gen_data_daily.get_date scheduled__2025-03-14T22:00:00+00:00 [queued]>
[2025-03-17T17:47:39.996+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 2
[2025-03-17T17:47:40.008+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): get_date> on 2025-03-14 22:00:00+00:00
[2025-03-17T17:47:40.015+0000] {standard_task_runner.py:60} INFO - Started process 532 to run task
[2025-03-17T17:47:40.019+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'gen_data_daily', 'get_date', 'scheduled__2025-03-14T22:00:00+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/gen_data_daily.py', '--cfg-path', '/tmp/tmpl5tyv_1x']
[2025-03-17T17:47:40.020+0000] {standard_task_runner.py:88} INFO - Job 16: Subtask get_date
[2025-03-17T17:47:40.091+0000] {task_command.py:423} INFO - Running <TaskInstance: gen_data_daily.get_date scheduled__2025-03-14T22:00:00+00:00 [running]> on host d107d9d70093
[2025-03-17T17:47:40.196+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='longnv' AIRFLOW_CTX_DAG_ID='gen_data_daily' AIRFLOW_CTX_TASK_ID='get_date' AIRFLOW_CTX_EXECUTION_DATE='2025-03-14T22:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-14T22:00:00+00:00'
[2025-03-17T17:47:40.198+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:315: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2025-03-17T17:47:40.198+0000] {python.py:201} INFO - Done. Returned value was: 2025-03-15
[2025-03-17T17:47:40.233+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=gen_data_daily, task_id=get_date, execution_date=20250314T220000, start_date=20250317T174739, end_date=20250317T174740
[2025-03-17T17:47:40.285+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-17T17:47:40.315+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
